src/parser/core/modules/About.tsx

```diff
@@ -25,6 +29,7 @@ export default class About extends Analyser {
 	//		from: ...,
 	//		to: ...,
 	// }
+	supportedLevels = null
```

Comment:
hm. i'm not sure how to feel about explicitly mentioning "support". iiuc, we're not really trying to support synced content, but rather making sure synced content isn't missing data completely in the timeline.

Reply:
Removed the supportedLevels support range.